src/Sylius/Behat/Context/Setup/OrderContext.php

```diff
@@ -864,16 +873,11 @@ private function createOrder(
         return $order;
     }
 
-    /**
-     * @param string|null $localeCode
-     *
-     * @return OrderInterface
-     */
     private function createCart(
         CustomerInterface $customer,
         ChannelInterface $channel = null,
```

Comment:
```suggestion
        ?ChannelInterface $channel = null,
```

We need to force this approach with CS, as both used all over the codebase.

Reply:
https://github.com/Sylius/Sylius/pull/15876
Here's the PR 🕺